src/test/java/com/google/jenkins/plugins/computeengine/integration/ITUtil.java

```diff
@@ -112,10 +129,30 @@
       String.format("%s@%s.iam.gserviceaccount.com", System.getenv("GOOGLE_SA_NAME"), PROJECT_ID);
   private static final String RETENTION_TIME_MINUTES_STR = "";
   private static final String LAUNCH_TIMEOUT_SECONDS_STR = "";
-  static final String SSH_USER = "test-user";
-  private static final GoogleKeyPair SSH_KEY = GoogleKeyPair.generate(SSH_USER);
+  static final int SNAPSHOT_TIMEOUT = windows ? 600 : 120;
+  private static final GoogleKeyPair SSH_KEY = GoogleKeyPair.generate(RUN_AS_USER);
   static final String SSH_PRIVATE_KEY = SSH_KEY.getPrivateKey();
+  private static final String WINDOWS_STARTUP_SCRIPT =
```

Comment:
Would it make sense to de-dup these startup scripts with their counterparts in the launchers? Same question with the SSH setup logic here.